### DISCUSSION TITLE
Is there intent to add in chips?

### DISCUSSION THREAD

**Discussion by omni-nathan** [uswds/uswds#5412] [Ideas]
Howdy! I was looking for this in the package and was surprised not to find it. Could this be added or is there new research saying this isn't a good pattern?
https://m3.material.io/components/chips/overview
<img width="1071" alt="image" src="https://github.com/uswds/uswds/assets/124096962/74600d34-a1cd-4d73-bb64-eb5ec682475e">

**Comment by sarah-sch** [comment]
Hi, @omni-nathan. Thanks for brining this up, and sorry for the slow. reply. I will let others speak to the pros and cons of the chips pattern. But I did want to encourage you to attend the USWDS February monthly call on Feb. 15 at 2:00 p.m. It will focus on the USWDS component lifecycle, including the process for proposing new components.
